recipes-core/psplash/psplash_%.bbappend

```diff
@@ -0,0 +1,17 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += " \
+    file://psplash-start.service \
+    file://psplash-stop.service \
+"
+
+inherit systemd
+
+SYSTEMD_PACKAGES = "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', '${PN}', '', d)}"
+SYSTEMD_SERVICE_${PN}_append = "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', ' psplash-start.service', '', d)}"
+
+do_install_append () {
+    install -d ${D}${systemd_unitdir}/system
+    install -m 644 ${WORKDIR}/psplash-start.service ${D}${systemd_unitdir}/system
```

Comment:
consider to use `${systemd_system_unitdir}` instead of `${systemd_unitdir}/system`